### ISSUE TITLE
Astropy 6.0 support

### ISSUE THREAD

**Issue by Theodlz** [astropy/astroplan#580]
Hi! The repo seems unactive for a couple of months now, and it looks like new versions of astropy don't work with astroplan. I've been trying to run the latest astropy (version >6), and astroplan looks for a configuration thing that doesn't exist anymore:

```bash
File "/home/runner/work/skyportal/skyportal/skyportal/facility_apis/growth_india.py", line 5, in <module>
    from astroplan import Observer, is_always_observable
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/astroplan/__init__.py", line 18, in <module>
    from ._astropy_init import *
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/astroplan/_astropy_init.py", line 21, in <module>
    from astropy.config.configuration import (
ImportError: cannot import name 'update_default_config' from 'astropy.config.configuration' (/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/astropy/config/configuration.py)
```

Any plans on supporting astropy 6 and above? I'm happy to try to contribute if necessary

**Comment by bmorris3** [comment]
Thanks for the heads up. I'll fix this ASAP. 👌🏻

**Comment by Theodlz** [comment]
That was quick!!! Thank you so much, I really appreciate it. Btw we use astroplan in https://github.com/skyportal/skyportal.

**Comment by bmorris3** [comment]
No problem. And that's awesome, I love seeing it in the wild! I just released astroplan v0.10 with the fix. Thanks again.

**Comment by Theodlz** [comment]
Thank you so much !!! 🚀 🚀 🚀 🚀 🚀
